ui/lib/apps/Statement/pages/List/index.tsx

```diff
@@ -181,11 +199,18 @@ export default function StatementsOverview() {
             )}
             {enable && (
               <Tooltip title={t('statement.pages.overview.toolbar.refresh')}>
-                {loadingStatements ? (
-                  <LoadingOutlined />
-                ) : (
-                  <ReloadOutlined onClick={refresh} />
-                )}
+                <Tooltip
```

Comment:
Will user just see 2 tooltips when hovering?

Reply:
That's right, maybe I should hide the refresh one when the outdated tooltip is popped up.